examples/semantic_segmentation.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>Maybe add a comment along the lines of:</p><p> </p><p>"We'll explicitly define a random seed to use when calling <code>ds.shuffle</code> to ensure our results are the same each time we run this cell."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>